demisto_sdk/commands/common/tools.py

```diff
@@ -965,6 +965,7 @@ def get_dict_from_file(path: str, use_ryaml: bool = False) -> Tuple[Dict, Union[
 
 
 # flake8: noqa: C901
+@lru_cache()
```

Comment:
what happens when I run on several files?

Reply:
it'll cache the result in the memory. if it's for different files, it'll store a map of input -> output. 
Least Recently Used - Will delete things that are not in use.